Lib/fontTools/varLib/instancer.py

```diff
@@ -150,25 +150,29 @@ def setMvarDeltas(varfont, deltaArray):
         tableTag, itemName = MVAR_ENTRIES[mvarTag]
         varDataIndex = rec.VarIdx >> 16
         itemIndex = rec.VarIdx & 0xFFFF
-        deltaRow = deltaArray[varDataIndex][itemIndex]
-        delta = sum(deltaRow)
+        delta = deltaArray[varDataIndex][itemIndex]
         if delta != 0:
             setattr(
                 varfont[tableTag],
                 itemName,
-                getattr(varfont[tableTag], itemName) + otRound(delta),
+                getattr(varfont[tableTag], itemName) + delta,
```

Comment:
seems worth commenting why we do/do not round

Reply:
because we have already summed and rounded them in _mergeVarDataItemColumns